test/scripts/extend/helper.js

```diff
@@ -31,18 +34,45 @@ describe('Helper', () => {
   });
 
   it('list()', () => {
-    const h = new Helper();
-
     h.register('test', () => {});
 
     h.list().should.have.keys(['test']);
   });
 
   it('get()', () => {
-    const h = new Helper();
-
     h.register('test', () => {});
 
     h.get('test').should.exist;
   });
+
+  it('setProp()', () => {
+    // name prop
+    h.setProp('foo', 'bar');
+
+    h.prop.foo.should.eql('bar');
+
+    // no name
+    try {
+      h.setProp();
```

Comment:
Recently I found `should.fail()` is necessary in `try...catch`. see https://github.com/hexojs/hexo-fs/pull/51#discussion_r364247814

Reply:
Added.